kadai/settings.py

```diff
@@ -76,7 +78,7 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': BASE_DIR / 'db.sqlite3',
+        'NAME': str(BASE_DIR / "db.sqlite3"),
```

Comment:
- [ ] ここはひとまずこれで直す必要はないと思われます（もし意図があれば教えてください）
```suggestion
        'NAME': BASE_DIR / "db.sqlite3",
```